src/components/transactions/UntrustedTxWarning/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { Tooltip, SvgIcon, Box } from '@mui/material'
+import WarningIcon from '@/public/images/notifications/warning.svg'
+
+const UntrustedTxWarning = ({ trusted }: { trusted: boolean }) => {
+  if (trusted) {
+    return null
+  }
+
+  return (
+    <Box data-testid="warning" gridArea="nonce">
```

Comment:
I would suggest keeping this wrapper outside of this component i.e. in the `TxSummary` since its specific to that grid layout.